angular-portfolio/src/app/award-detail/award-detail.component.html

```diff
@@ -0,0 +1,16 @@
+<div *ngIf="award">
+    <div class="flex-container">
+        <div *ngFor="let awardloop of awards" 
```

Comment:
Rename the original "award" to "awardYear" or something, and make this simply "let award of awards".